mappings/net/minecraft/block/BubbleColumnBlock.mapping

```diff
@@ -31,7 +31,7 @@ CLASS bgw net/minecraft/block/BubbleColumnBlock
 		ARG 6 neighborPos
 	METHOD a appendProperties (Lbpn$a;)V
 		ARG 1 builder
-	METHOD b getBucketFluid (Lbbq;Let;Lbpm;)Lcfc;
+	METHOD b tryDrainFluid (Lbbq;Let;Lbpm;)Lcfc;
```

Comment:
Why not just `drainFluid`? I realise it's an action which can fail, but the `try` prefix feels a little superfluous.

Reply:
`FluidFillable` initially had `fillWithFluid` which then changed to `tryFillWithFluid`, so with the `try` prefix, the naming is consistent between the interfaces.